### PR TITLE
Fuzzy Search (Independent Publisher)

### PR DESCRIPTION
**Pull request by mitanshug**
console-test

pic2

pic3

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/Fuzzy Search/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  },
+  "host": "sa018.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata":[
+    {
+      "propertyName":"Website",
+      "propertyValue":"https://sa018.azurewebsites.net"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Fuzzy Search/apiDefinition.swagger.json:21]: @happyume this should be a valid URL to the product.

File: independent-publisher-connectors/Fuzzy Search/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    },
+    {
+      "propertyName":"Privacy Policy",
+      "propertyValue":"https://sa018.azurewebsites.net"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Fuzzy Search/apiDefinition.swagger.json:25]: @happyume please replace with a valid URL to the privacy policy page.

File: independent-publisher-connectors/Fuzzy Search/readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## Fuzzy Search
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Fuzzy Search/readme.md:1]: @happyume please include all required section in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).

File: independent-publisher-connectors/Fuzzy Search/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+   "properties": {
+     "connectionParameters": {},
+     "iconBrandColor": "#da3b01",
+     "capabilities": [
+       "actions"
+     ],
+     "publisher": "Mitanshu Garg",
+     "stackOwner": "Mitanshu Garg"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Fuzzy Search/apiProperties.json:9]: @happyume stackOnwer should be the API Owner. 
"stackOwner": "Name of company or organization that connector is connecting to."

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@happyume you are adding the same file twice. in the root folder and in the independent publisher connectors folder. Please remove the files from the root of this repo.
image

**Comment by Amjed-Ayoub** [comment]
Hello @happyume ,
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by mitanshug** [comment]
Hi
Thanks for the follow-up.

I am on holiday till end of May. I do have a few questions and will get
back to you after the holidays when I have access to my laptop and account.

Do you think it will be possible to keep my pull request open till that
time?

Regards
Mitanshu

**Comment by vmanoharas** [comment]
Hello @happyume 
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by vmanoharas** [comment]
Hello @happyume, 

This PR is in open state from many months. I am closing this PR, kindly reopen/create new PR, if you have any progress.

**Pull request closed without merging**